fastify.js

```diff
@@ -893,8 +893,6 @@ function fastify (options) {
   }
 }
 
-fastify.errorCodes = errorCodes
```

Comment:
This is a breaking change.

Reply:
Actually, Not.

```js
fastify.errorCodes = errorCodes
module.exports = fastify
// is the same as 
module.exports = fastify
module.exports.errorCodes = errorCodes
```

It is all modifying the same `fastify` object.

Reply:
And currently importing the errorCodes in the current implementation is broken when using ESM.